source/mixin/network_mixin.py

```diff
@@ -0,0 +1,40 @@
+from PyQt4.QtCore import *
+from PyQt4.QtNetwork import *
+
+
+class NetworkMixin(QNetworkAccessManager):
+    request_url = str
+    result_connection = ""
+
+    @pyqtSlot()
+    def connectionFinished(self):
+        print(self.result_connection)
+
+    @pyqtSlot()
+    def connectionReadData(self):
+        self.result_connection += self.reply.readAll()
+
+    def connect(self):
+        print "connect it"
+        QObject.connect(
+            self.reply, SIGNAL('readyRead()'),
+            self, SLOT('connectionReadData()'))
+
+        QObject.connect(
+            self.reply, SIGNAL('finished()'),
+            self, SLOT('connectionFinished()'))
+
+    def connectGET(self):
```

Comment:
Also I don't know if it is good to overload the reserved method name 'connect' it might create side effects